### PR TITLE
Scans: added non default info

### PR DESCRIPTION
**Pull request by psiinon**
Just for Firing Range to start with, if everyones good with this I can add them to the other scan pages as well

Screenshot 2022-08-25 at 14-25-07 OWASP ZAP – ZAP vs Firing Range

Signed-off-by: Simon Bennetts <psiinon@gmail.com>

### CODE REVIEW

File: site/content/docs/scans/firingrange.md [hunk 1]
```diff
@@ -31,8 +31,17 @@ Changes which find any of the missed vulnerabilities are eligible for a bounty v
...
 | Frequency | Daily |
 | Scripts | https://github.com/zapbot/zap-mgmt-scripts/blob/master/scans/firingrange/ |
 | Action | https://github.com/zapbot/zap-mgmt-scripts/actions/workflows/zap-vs-firingrange.yml | 
+
+
+#### Non Default Settings
+
+The latest Live ZAP release is run with the default setting against this app with the following exceptions:
+
+* The [XSS](/docs/alerts/40012/) rule is set to use LOW threshold in order to detect 2 cases which are not strictly vulnerable
```
Comment (kingthorin) [inline site/content/docs/scans/firingrange.md:47]: I'd suggest a period here.

File: site/content/docs/scans/firingrange.md [hunk 2]
```diff
@@ -31,8 +31,17 @@ Changes which find any of the missed vulnerabilities are eligible for a bounty v
...
+#### Configuration
+
 | Config | Details |
 | --- | --- |
 | Frequency | Daily |
 | Scripts | https://github.com/zapbot/zap-mgmt-scripts/blob/master/scans/firingrange/ |
 | Action | https://github.com/zapbot/zap-mgmt-scripts/actions/workflows/zap-vs-firingrange.yml | 
+
+
+#### Non Default Settings
```
Comment (kingthorin) [inline site/content/docs/scans/firingrange.md:43]: For some reason in your screenshot this seems quite squished together, might be an explicit </br> or </p> :man_shrugging:

Comment (psiinon) [reply site/content/docs/scans/firingrange.md:43]: Its CSS, I'll try to fix that, but its not my strong suit :/

**Comment by psiinon** [comment]
Think I've fixed that with a hack ;)

Screenshot 2022-08-25 at 15-59-03 OWASP ZAP – ZAP vs Firing Range

**Review by thc202 — APPROVED** [review]

**Comment by thc202** [comment]
Thank you!

**Review by kingthorin — APPROVED** [review]

**Pull request merged**